sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -159,6 +161,8 @@ Console.WriteLine($"    Negative confidence score: {docSentiment.ConfidenceScore
 ```
 For samples on using the production recommended option `AnalyzeSentimentBatch` see [here][analyze_sentiment_sample].
 
+To get more granular information about the opinions related to aspects of a product/service, also knows as Aspect-based Sentiment Analysis in Natural Language Processing (NLP), see sample on sentiment analysis with opinion mining [here][analyze_sentiment_opinion_mining_sample].
```

Comment:
knows -> known
see sample -> see a sample

Reply:
Thank you! 
Will fix in PR https://github.com/Azure/azure-sdk-for-net/pull/14871